# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/thunderbird/hotfix-glibc-2.43.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- firefox-147.0.1/security/sandbox/chromium/sandbox/linux/system_headers/linux_seccomp.h.vanilla	2026-01-25 12:31:42.631174351 +0100
+++ firefox-147.0.1/security/sandbox/chromium/sandbox/linux/system_headers/linux_seccomp.h	2026-01-25 12:35:13.031188681 +0100
@@ -215,7 +215,7 @@
 #endif
 
 #ifndef SYS_SECCOMP
-#define SYS_SECCOMP                   1
+//#define SYS_SECCOMP                   1
 #endif
 
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
